torch/_C/__init__.pyi.in

```diff
@@ -413,6 +419,11 @@ class PyTorchFileWriter(object):
     def write_end_of_file(self) -> None: ...
     ...
 
+def _jit_get_inline_everything_mode() -> _bool: ...
+def _jit_set_inline_everything_mode(enabled: _bool) -> None: ...
+def _jit_pass_dce(Graph) -> None: ...
+def _jit_pass_lint(Graph) -> None: ...
```

Comment:
can you move these together with the rest of the `def _jit_*` ?

Reply:
I added them here bcoz they fall under "# Defined in torch/csrc/jit/python/init.cpp".

Reply:
good catch. didn't see that since github only shows +/- 5 lines